src/validators/ClassValidator.ts

```diff
@@ -44,10 +45,11 @@ export class BsClassValidator {
         return cls;
     }
 
+
     /**
-     * Find all "new" statements in the program,
-     * and make sure we can find a class with that name
-     */
+    * Find all "new" statements in the program,
+    * and make sure we can find a class with that name
```

Comment:
The jsdoc comment style requires all of the stars to be at the same indentation level. The previous code was actually correct, so can you undo this chunk?